modal/functions.py

```diff
@@ -107,7 +107,14 @@ def __init__(self, stub: api_grpc.ModalClientStub, function_call_id: str, client
         self.function_call_id = function_call_id  # TODO: remove and use only input_id
 
     @staticmethod
-    async def create(function: "_Function", args, kwargs, *, client: _Client) -> "_Invocation":
+    async def create(
+        function: "_Function",
+        args,
+        kwargs,
+        *,
+        client: _Client,
+        function_call_invocation_type: "api_pb2.FunctionCallInvocationType.ValueType",
```

Comment:
I believe quotes are mandatory for `if TYPE_CHECKING` gated imports, but not needed here.